cli/valet.php

```diff
@@ -181,6 +181,35 @@
         info('The ['.$url.'] site will now serve traffic over HTTP.');
     })->descriptions('Stop serving the given domain over HTTPS and remove the trusted TLS certificate');
 
+    /**
+     * Create an Nginx proxy config for the specified domain
+     */
+    $app->command('proxy domain host', function ($domain, $host) {
+
+        Site::proxyCreate($domain, $host);
```

Comment:
I'm not sure where the host-must-be-a-valid-URL-ish-thing validation should happen. Here, or inside `Site::proxyCreate`?

I used the following inside `Site::proxyCreate` and it prevented me from entering hosts like `bar`, `/bar`, and `b/a/r`:

```
if (! preg_match('~^http(?:s|)://.*$~', $host) ) {
    throw new \InvalidArgumentException(sprintf('"%s" is not a valid URL', $host));
}
```

```
$ valet proxy foo bar

In Site.php line 643:

  "bar" is not a valid URL


proxy <domain> <host>
```

Reply:
Agreed.